translations/ui-checkin/en_US.json

```diff
@@ -84,7 +84,7 @@
     "confirmModal.confirm": "Confirm",
     "confirmModal.cancel": "Cancel",
     "statuses.missing": "Missing",
-    "statuses.declaredLost": "'Declared lost",
+    "statuses.declaredLost": "Declared lost",
```

Comment:
It looks like this has seeped out to `en_GB`, `it`, et al, so I think the right place to correct is actually in `en.json` in order to trigger other locales to make the same update. That said, for the purposes of making folio-snapshot look good while we wait for this fix to get pushed out to Lokalise and then get pulled back in via PRs, updating `en_US.json` as you did here is also worthwhile.

Reply:
Thanks for catching that! I meant to make the change in `en.json` and didn't notice I'd opened the wrong file.